packages/guides/src/ReferenceResolvers/Interlink/DefaultInventoryLoader.php

```diff
@@ -46,11 +48,22 @@ public function loadInventoryFromJson(Inventory $inventory, array $json): void
             if (is_array($groupArray)) {
                 foreach ($groupArray as $linkKey => $linkArray) {
                     if (!is_array($linkArray) || count($linkArray) < 4) {
+                        $this->logger->warning('Invalid Inventory entry found. ');
+                        continue;
+                    }
+
+                    if (!is_string($linkArray[2])) {
+                        $this->logger->warning('Invalid Inventory entry found. ');
+                        continue;
+                    }
+
+                    if (!is_scalar($linkArray[0] ?? '') || !is_scalar($linkArray[1] ?? '') || !is_scalar($linkArray[3] ?? '')) {
+                        $this->logger->warning('Invalid Inventory entry found. ');
```

Comment:
Maybe we should emit unique errors for each string here, so that later we easier know what exact error is triggered?

Reply:
Done